AirCasting/Graph/StatisticsContainer/ViewModel/StatisticsContainerViewModelable.swift

```diff
@@ -11,7 +11,7 @@ struct SingleStatViewModel: Identifiable {
 
     let id: Int
     let title: String
-    let value: String
+    let value: Double
```

Comment:
The viewmodel should provide already formatted stuff for the UI. Views should only be responsible for the layout, having all provided by the viewmodel

Reply:
Added a String value, then :)